quickwit-cli/tests/prepare_tests.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+awslocal s3 mb s3://quickwit-integration-tests && awslocal s3 rm --recursive s3://quickwit-integration-tests
```

Comment:
I think rm is not necessary as we don't use localstack persistence

Reply:
if something goes wrong it's necessary

Reply:
and this works only if localstack is started

Reply:
ok, locally it is necessary as it's annoying to stop and restart localstack

Reply:
in the localstack api I see only a start command, no stop command